packages/shared/lib/core/layer-2/tests/parseLayer2MetadataForTransfer.test.ts

```diff
@@ -21,6 +21,25 @@ describe('Function: parseLayer2MetadataForTransfer.ts', () => {
         expect(parsedMetadata).toEqual(expected)
     })
 
+    it('should correctly parse metadata with long base token', () => {
```

Comment:
In this PR https://github.com/iotaledger/firefly/pull/7702 we already had the bigint fix so this test was added too with some improvements like the ethereum address shouldnt be as large, I think this test can be removed as the one in `develop` is correct already 🙏🏼